lwt.opam

```
@@ -31,7 +31,7 @@ depends: [
 depopts: ["base-threads" "base-unix" "conf-libev"]
 dev-repo: "git+https://github.com/ocsigen/lwt.git"
 build: [
-  ["dune" "subst"] {dev}
+  ["dune" "subst"] {pinned}
```

afaik, this should always be

```suggestion
  ["dune" "subst"] {dev}
```

see https://github.com/ocaml/dune/pull/3647

---

dune 3.0 seems to fix that.

---

Nice. Thanks :)